package.json

```diff
@@ -39,9 +39,10 @@
     "pg": "8.7.3",
     "pg-query-stream": "^4.2.3",
     "raw-body": "^2.5.1",
-    "warp-contracts": "1.2.13",
+    "warp-contracts": "1.2.24",
```

Comment:
need to upgrade when this one is installed https://github.com/warp-contracts/warp/pull/257